### PR TITLE
✚ Added iotwifi - AP + Client wifi management. 

### PR DESCRIPTION
**Pull request by cjimti**
Wifi management docker container for Pi 3.

# Please make sure all below checkboxes have been checked before submitting your PR.

- [x] I have read and understood the [contribution guidelines](https://github.com/thibmaek/awesome-raspberrypi/blob/master/CONTRIBUTING.md).
- [x] This pull request has a descriptive title. *(For example: `Add Raspbian`)*
- The topic I added
	- [x] includes a valid (https) link,
	- [x] includes a concise and on-topic description,
	- [x] mentions if there is only support some devices,
	- [x] is not a duplicate,
	- [x] has been added at the bottom of the appropriate category,
	- [x] is in the form of **Named Link - Description, seperated by commas.**
	- [x] ends with a dot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ The complete collection of Raspberry Pi models consist of:
 - [CusDeb](https://cusdeb.com/en/) - Web UI for creating custom images with specific packages installed.
 - [Alpha](https://github.com/farjump/raspberry-pi) - Remotely load, debug and test bare-metal programs using GDB with this system-level GDB server.
 - [Pieman](https://github.com/tolstoyevsky/pieman) - Script for creating custom images based on Raspbian, Devuan and Ubuntu.
+- [PiFi (AP + Client)](https://github.com/cjimti/iotwifi) - 8MB [Docker Container](https://hub.docker.com/r/cjimti/iotwifi/) with REST API for controlling wifi client and station modes simultaneously. ![Supports Raspberry Pi 3](/media/badges/rpi-3.png)
```
Comment (thibmaek) [inline README.md]: This project does not seem to be called PiFi when I check its Github page, it shows iotwifi instead. Could you correct the title, please?

Comment (thibmaek) [inline README.md]: This project does not seem to be called PiFi when I check its Github page, it shows iotwifi instead. Could you correct the title, please?

**Comment by cjimti** [comment]
No problem. At one point I was calling it pifi, but changed the name since I am working on supporting other devices as well, thanks for calling that out. 👍

**Comment by thibmaek** [comment]
Cool :)

**Pull request merged**